storm-shaded-deps/pom.xml

```diff
@@ -297,6 +297,28 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-pmd-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>compile</id>
```

Comment:
Maybe change the id to something like `<id>workaround-makeItVisibleOnIntellij</id>` since the ID here is only informational, to better explain the purpose.

Reply:
Will test this change.